src/dune/dep.ml

```diff
@@ -1,9 +1,16 @@
 open Stdune
 
 module Trace = struct
+  module Fact = struct
+    type t =
+      | Env of (string * string option)
```

Comment:
Why do we add the parens around the constructors here?

Reply:
It was convenient for `File` because we construct if from function call result. It's not needed for `Env` and `File_selector` though.